Sources/Fullscreen/FavoriteAdsList/Subviews/FavoriteAdsSortingView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+class FavoriteAdsSortingView: UIView {
+
+    // MARK: - Internal properties
+
+    internal var title: String = "" {
```

Comment:
No need for `internal`?

Reply:
Nope, was trying to be explicit. Will remove